File: recaudacion/2.Codigo/frontend/src/app/core/models/liquidacion.ts
import { ClasificadorIngreso } from "./clasificadoringreso";
import { FuenteFinanciamiento } from "./fuentefinanciamiento";
import { TipoCaptacion } from "./tipocaptacion";

export class Liquidacion {
    liquidacionId: number;
    unidadEjecutoraId: number;
    tipoDocumentoId: number;
    fuenteFinanciamientoId: number;
    fuenteFinanciamiento: FuenteFinanciamiento;
    clienteId: number;
    reciboIngresoId: number;
    cliente: any;
    cuentaCorrienteId: number;
    cuentaCorriente: any;
    numero: string;
    procedencia: string;
    fechaRegistro: any;
    total: number;
    factura: string;
    boletaVenta: string;
    estado: number;
    estadoNombre: string;
    liquidacionDetalle: LiquidacionDetalle[];
    usuarioCreador: string;
    usuarioModificador: string;
    index: number;
}
export class LiquidacionEstado {
    liquidacionId: number;
    estado: number;
    usuarioModificador: string;
}
export class LiquidacionDetalle {
    clasificadorIngresoId: number;
    clasificadorIngreso: ClasificadorIngreso;
    tipoCaptacionId: number;
    tipoCaptacion: TipoCaptacion;
    importeParcial: number;
    usuarioCreador: string;
    usuarioModificador: string;
    reciboIngresoDetalle: any;
    index: number;
}

export class LiquidacionFilter {
    pageNumber: number;
    pageSize: number;
    sortColumn: string;
    sortOrder: string;
    unidadEjecutoraId: number;
    numero: string;
    estado: number;
    rol: string;
}

File: recaudacion/2.Codigo/backend/RecaudacionApiLiquidacion/Domain/Liquidacion.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace RecaudacionApiLiquidacion.Domain
{
    [Table("LIQUIDACION_RECAUDACION")]
    public class Liquidacion
    {
        [Key]
        [Column("LIQUIDACION_RECAUDACION_ID")]
        public int LiquidacionId { get; set; }
        [Column("UNIDAD_EJECUTORA_ID")]
        public int UnidadEjecutoraId { get; set; }
        [Column("FUENTE_FINANCIAMIENTO_ID")]
        public int FuenteFinanciamientoId { get; set; }
        [NotMapped]
        public FuenteFinanciamiento FuenteFinanciamiento { get; set; }
        [Column("TIPO_DOCUMENTO_ID")]
        public int TipoDocumentoId { get; set; }
        [Column("CLIENTE_ID")]
        public int ClienteId { get; set; }
        [Column("CUENTA_CORRIENTE_ID")]
        public int CuentaCorrienteId { get; set; }
        [Column("RECIBO_INGRESO_ID")]
        public int? ReciboIngresoId { get; set; }
        [Column("LIQUIDACION_RECAUDACION_NUMERO")]
        public string Numero { get; set; }
        [Column("LIQUIDACION_RECAUDACION_PROCEDENCIA")]
        public string Procedencia { get; set; }
        [Column("LIQUIDACION_RECAUDACION_FECHA")]
        public DateTime FechaRegistro { get; set; }
        [Column("LIQUIDACION_RECAUDACION_TOTAL", TypeName = "decimal(12,2)")]
        public decimal Total { get; set; }
        [Column("LIQUIDACION_RECAUDACION_FACTURA")]
        public string Factura { get; set; }
        [Column("LIQUIDACION_RECAUDACION_BOLETA")]
        public string BoletaVenta { get; set; }
        [Column("LIQUIDACION_RECAUDACION_ESTADO")]
        public int Estado { get; set; }
        [NotMapped]
        public string EstadoNombre { get; set; }
        [Column("USUARIO_CREADOR")]
        public string UsuarioCreador { get; set; }
        [Column("FECHA_CREACION")]
        public DateTime FechaCreacion { get; set; }
        [Column("USUARIO_MODIFICADOR")]
        public string UsuarioModificador { get; set; }
        [Column("FECHA_MODIFICACION")]
        public DateTime? FechaModificacion { get; set; }
    }

}
